### PR TITLE
Refactor second solution

### PR DESCRIPTION
**Pull request by giovanizanetti**
- Implement i18-react
- Rename single char variables meaningfully
- Refactor hooks bypassing necessary dependencies
- Fix typo.

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "home": "Home"
+  },
+  "pokemon_table_headings": {
+    "id": "ID",
+    "name": "Name",
+    "height": "Height",
+    "weight": "Weight",
+    "type": "Type"
+  },
+  "perpage_dropdown_labels": {
```
Comment (michaeltomasik) [inline public/locales/en/translation.json:12]: for numbers we don't put it there, just regular const file

File: src/tests/PokeList.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { Suspense } from 'react'
+
+describe('testSuite', () => {
+  // beforeEach(() => {
+
+  // })
+  test('should component display a table containing ten pokemon raws', async () => {
+    render(<PokeList />)
+
+    await screen.debug()
```
Comment (michaeltomasik) [inline src/tests/PokeList.spec.js:14]: this test is not testing anything

**Review by michaeltomasik — COMMENTED** [review]
Well done, just add tests proper tests and it will be alright :)

File: src/tests/PokeList.spec.js (hunk 2 above)
Comment (giovanizanetti) [reply src/tests/PokeList.spec.js:14]: I messed up my commit. The tests were not supposed to be committed yet.

**Pull request merged**